src/libraries/Microsoft.PowerFx.Interpreter/Functions/RuntimeHelpers.cs

```diff
@@ -31,5 +32,21 @@ public static StringValue ConcatString(IRContext irContext, StringValue arg1, St
             var str = string.Concat(arg1.Value, arg2.Value);
             return new StringValue(irContext, str);
         }
+
+        public static SymbolContext GetSymbolContext(EvalVisitorContext context, DValue<RecordValue> row)
```

Comment:
>GetSymbolContext [](http://example.com/codeflow?start=36&length=16)

With a new WithScopeValues(FormulaValue  scopeValues) helper, this could be pushed down.